src/libOpenImageIO/imagebufalgo_draw.cpp

```diff
@@ -298,7 +298,7 @@ noise_uniform_ (ImageBuf &dst, float min, float max, bool mono,
     // Serial case
     for (ImageBuf::Iterator<T> p (dst, roi);  !p.done();  ++p) {
         int x = p.x(), y = p.y(), z = p.z();
-        float n;
+        float n = 0.0;
         for (int c = roi.chbegin;  c < roi.chend;  ++c) {
             if (c == 0 || !mono)
```

Comment:
Ha, I just noticed, the 

    if (c == 0 || !mono)

should actually be

    if (c == roi.chbegin || !mono)

I will fix this when I merge your changes.